cmake_scripts/FindArrow.cmake

```diff
@@ -107,9 +127,8 @@ if (ARROW_INCLUDE_DIR AND ARROW_LIBS)
   else()
     set(ARROW_STATIC_LIB ${ARROW_PYTHON_LIB_PATH}/libarrow.a)
     set(ARROW_PYTHON_STATIC_LIB ${ARROW_PYTHON_LIB_PATH}/libarrow_python.a)
```

Comment:
still running into the `fatal error: arrow/python/pyarrow.h: No such file or directory` issue

Reply:
resolved by adding in the PYARROW_INCLUDE_DIR and calling it in the include_directories call in cpp/turbodbc_arrow/CMakeLists.txt